app/views/organized/manage/index.html.haml

```diff
@@ -40,29 +43,30 @@
   - if current_organization.is_paid
     .form-group.row.avatar-upload-wrapper
       %label.org-form-label
-        Organization logo
+        = t(".organization_logo")
       .col-sm-4
         - if @organization.avatar?
           = image_tag(@organization.avatar_url(:thumb))
-        / - else
-        /   %p
-        /     No picture!
+
         %label.file
           = f.file_field :avatar, class: 'avatar-upload-field'
           %span.file-custom
-            %span.file-upload-text Choose file...
+            %span.file-upload-text= t(".choose_file")
           = f.hidden_field :avatar_cache
 
   .form-group.row.mt-4
     .form-submit-button
-      = f.submit 'Update', class: 'btn btn-success btn-lg'
-
+      = f.submit t(".update"), class: 'btn btn-success btn-lg'
 
 - unless current_organization.is_paid
   %hr.delete-organization-section
   %p.text-md-center
     %a.collapsed{ href: '#', data: { toggle: 'collapse', target: '#delete-organization' } }
-      I would like to terminate my account with Bike Index
+      = t(".i_would_like_to_terminate_my_account")
   #delete-organization.collapse
     .text-md-center
-      = link_to 'Delete organization', organization_manage_path(id: current_organization.id, organization_id: current_organization.to_param), method: :delete, data: { confirm: "Are you positive you want to remove #{current_organization.name} from Bike Index?" }, class: 'btn btn-danger'
+      = link_to t(".delete_organization"),
+      organization_manage_path(id: current_organization.id, organization_id: current_organization.to_param),
```

Comment:
Should this be indented one level deeper?

Reply:
Apparently it works either way (so long as the break is at a comma) but it looks a bit cleaner aligned with the `link_to`, so I updated it to that 👍

Reply:
So many learnings on this PR!